common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -2,22 +2,20 @@
     <div class="indicator_container">
     % if input_type == 'checkbox' or not value:
         <span class="status ${status.classname if show_correctness != 'never' else 'unanswered'}"
-            id="status_${id}"
-            aria-describedby="inputtype_${id}">
+            id="status_${id}">
             <span class="sr">
                 %for choice_id, choice_description in choices:
                 % if choice_id in value:
                     ${choice_description}, 
                 %endif
                 %endfor
-                -
                 ${status.display_name}
             </span>
         </span>
     % endif
     </div>
 
-    <fieldset role="${input_type}group" aria-label="${label}">
+    <fieldset>
 
```

Comment:
@ahsan-ul-haq, it's very important for accessibility to include the value of ```${label}``` in a ```<legend>``` element, as a child of this ```<fieldset>```
```
<legend>${label}</legend>
```
Once we do this, the paragraph that is currently being used to display the question text, will not be necessary.  I understand that changing this will affect all question types.  Can we include a conditional statement during this transition so that the paragraph is not displayed in question types that have been converted to use the new accessible format?